bin/esy.re

```diff
@@ -233,6 +233,55 @@ let buildDependencies = (all, mode, pkgarg, proj: Project.t) => {
   Project.withPackage(proj, pkgarg, f);
 };
 
+let gc = (projCfgs: list(ProjectConfig.t), dryRun) => {
+  open RunAsync.Syntax;
+  let mode = BuildSpec.BuildDev;
+  let%bind (installedBuilds, allCachedBuilds) =
+    RunAsync.List.foldLeft(
+      ~init=(PathSet.empty, PathSet.empty),
+      ~f=
+        (acc, projCfg) => {
+          let (installedBuilds, allCachedBuilds) = acc;
+          let%bind (proj, _) = Project.make(projCfg);
+          let%bind plan = Project.plan(mode, proj);
+          let%bind ibs = Project.scanDependencies(proj, plan);
```

Comment:
What's `ibs`?

Reply:
Please ignore 😄  Renamed the variable to allProjectDependencies